lib/core/boot/boot.ts

```diff
@@ -68,11 +68,24 @@ export const bootstrap = (): FluxifyServer => {
 					}
 				}
 
+				start(request, 'auth');
+				let jwt: unknown | null = null;
```

Comment:
Does this need to be `unknown` if later gets assigned with a string? Isn't it possible to be `string | null`?

Reply:
It's actually not a `string`, the type would be `JwtDto`. The reason behind this is if you do not provide a `jwtDto` validation schema the type stays `unknown` because it can be anything. When you provide one it get's validated and the intellisense kicks in. fyi the param, query, and body objects work the same. You can find those on any route handler.